src/stories/components/Branding/styles.scss

```diff
@@ -0,0 +1,14 @@
+@import '../../../scss/variables.scss';
+
+// @file
+// Branding
+
+.branding {
+  font-size: $font-size-xl;
+  font-weight: bold;
```

Comment:
I couldn't find a variable for bold font-weight in the presets that Alessia added so I added this here. But if it would be better to factor that out for later reuse I can try to do that.

Reply:
I think it should be fine if the one you need is not defined in the global variable. If not, it should be a very easy fix.

Reply:
@fernandowinfield no variable for font weights. You can use `bold` or `700` to achieve that!